src/lesson03/Task08.java

```diff
@@ -0,0 +1,12 @@
+package lesson03;
+
+import java.util.Scanner;
+
+public class Task08 {
+    public static void main(String[] args) {
+        Scanner scan = new Scanner(System.in);
+        String n = scan.nextLine();
+        System.out.println(new StringBuilder(n).reverse().toString());
+
```

Comment:
Коли ти обєкт передаєш в метод print в консоль, то компілятор сам викличе метод toString()
```suggestion
        System.out.println(new StringBuilder(n).reverse());

```